### PR TITLE
Games actions

### PR DESCRIPTION
**Pull request by Toren242**

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -2,4 +2,33 @@ class GamesController < ApplicationController
   def index
     @games = Game.all
   end
+
+  def destroy
+    @game = Game.find(params[:id])
+    @user = Game.where(user: current_user)
```
Comment (cy-porteous) [inline app/controllers/games_controller.rb:8]: Needs an amendment, see following:

@game.user = current_user

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 <h1>Pages#home</h1>
+<h2><%= %></h2>
```
Comment (cy-porteous) [inline app/views/pages/home.html.erb:2]: Needs looking into, no HTML?

Comment (Toren242) [reply app/views/pages/home.html.erb:2]: done thanks

File: app/controllers/games_controller.rb (hunk 1 above)
Comment (Toren242) [reply app/controllers/games_controller.rb:8]: done

**Pull request merged**